pkg/kubevirt/machine.go

```diff
@@ -219,6 +240,80 @@ func (m *Machine) IsReady() bool {
 	return m.hasReadyCondition()
 }
 
+const (
+	defaultCondReason  = "VMNotReady"
+	defaultCondMessage = "VM is not ready"
+)
+
+func (m *Machine) GetVMUnscheduledReason() (reason string, message string) {
+	reason = defaultCondReason
+	message = defaultCondMessage
```

Comment:
I wonder if the default message here should include the vm's printable status? 

something like

message := fmt.Sprintf("%s: %s", defaultCondMessage, vm.Status.PrintableStatus)